CONTRIBUTING.POSTGRESQL.CHEATSHEET.md

```diff
@@ -0,0 +1,176 @@
+# Contributing: Cheatsheet (PostgreSQL)
+
+The [CONTRIBUTING](CONTRIBUTING.md) doc provides an extensive amount of information. Below is a summary of that info to get up-and-running quickly, with a few additional explanations.
+
+## Pre-Requisites
+
+1. Have docker installed
+1. Have `yarn` installed (see _Helpful Notes_ section below)
+1. Have the database installed and running
+
+   Test Type | Requires Running Database
+   ---         | --- |
+   Unit        | No  |
+   Integration | Yes |
+   Other       | Yes | 
+
+   Note:
+   - tests DO NOT automatically start the database they're testing against  
+   (see _Helpful Notes_ section below)
+   - if the database is not running, you may encounter an error message similar to:
+      ```bash
+      SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:23010
+      ```
+
+---
+
+## Running Tests
+This example will use _PostgreSQL_ as the database being contributed to. Replace _postgres_ with whatever dialect is specific to your changes.
+
+The basic structure is start, test, stop. Start the database, run the test script, stop the database.
+
+```bash
+yarn start-postgres            # start the database
+yarn test-integration-postgres # run the test
+yarn stop-postgres             # stop the database
+```
+
+### Different Tests
+
+Test | Description
+---- | ---
+`DIALECT=postgres yarn test` | **Type**: Full test of a database<br>**Requires**: specified dialect & running db
+`yarn test-integration` | **Type**: Partial test<br>**Requires**: running db
```

Comment:
Maybe explain the difference between integration and unit tests a bit more? Also; these also need the dialect